src/mpid/ch4/src/ch4_win.h

```diff
@@ -13,6 +13,30 @@
 
 #include "ch4_impl.h"
 
+static inline void MPIDI_create_workqs(MPIR_Win* win){
```

Comment:
I would prefer `MPIDI_win_create_workq` or something that implies this function is applied to windows.